scripts/dotnet.py

```diff
@@ -621,17 +621,20 @@ def get_commit_date(
         url = urlformat % (owner, repo, commit_sha)
 
     build_timestamp = None
+    item = None
```

Comment:
`item = None` and `retrycount = 0` lines can be removed as they are local to the for loop. Also to make things a bit nicer to read, you could move the for loop into a function `get_build_timestamp` and that function can just return the timestamp from inside the for loop, meaning no need to have a `build_timestamp = None` variable

Reply:
Followed the local variable guidance, glad you caught that! For the loop, I am going to keep it as is for now with potentially changing to match the function format you brought up in the future.